seed_services_cli/tests/test_stage_based_messaging.py

```diff
@@ -175,3 +181,40 @@ def test_message_import_csv(self, create_patch):
             'sequence_number': '2',
             'binary_content': '',
         })
+
+
+class TestSbmMessagesUpdate(TestStageBasedMessagingCommands):
+
+    def test_messages_update_help(self):
+        result = self.runner.invoke(cli, ['sbm-messages-update', '--help'])
+        self.assertEqual(result.exit_code, 0)
+        self.assertTrue(
+            "Update messages in the Stage Based Messaging service."
+            in result.output)
+
+    def test_message_update_error_with_no_param(self):
+        result = self.runner.invoke(cli, ['sbm-messages-update'])
+        self.assertEqual(result.exit_code, 2)
+        self.assertTrue(
+            'Please specify either --csv or --json.' in result.output)
+
+    @patch('seed_services_client.StageBasedMessagingApiClient.update_message')
+    def test_message_update_csv(self, create_patch):
+        csv_file = tempfile.NamedTemporaryFile()
+        csv_file.write(
+            b'messageset,sequence_number,lang,text_content,binary_content\n')
+        csv_file.write(b'1,2,eng_ZA,"message text",""')
+        csv_file.flush()
+
+        result = self.runner.invoke(
+            cli, ['sbm-messages-update', '--csv={0}'.format(csv_file.name)])
+        csv_file.close()
+
+        self.assertEqual(result.exit_code, 0)
+        create_patch.assert_called_with({
```

Comment:
This function should be called with the `id` that you put in the API lookup mock too.